### ISSUE TITLE
Backticks in SECTION name gets stripped out causing no tests to run

### ISSUE THREAD

**Issue by TheButlah** [matepek/vscode-catch2-test-adapter#140]
**Checklist** (You can delete this section, don't have to be the part of your issue.)

- [X] The issue is about this extension and NOT about a fork.
- [X] Check the [known issues](https://github.com/matepek/vscode-catch2-test-adapter#known-issues) list.
- [X] The latest version of the extension was used.

**Describe the bug** (A clear and concise description of what the bug is.)
When writing a catch2 test case that contains a backtick (other symbol on tilde key) in its name, for example, 

**To Reproduce** (Steps to reproduce the behavior.)

1. Make a c++ project that uses catch2, and create a new catch2 test file.
2. Put the following code in the test file:
```c++
#define CATCH_CONFIG_MAIN
#include <catch2/catch.hpp>

SCENARIO("`Config`s can be serialized") {
    REQUIRE(true==true);
}
```
3. Build the file
4. Try debugging the file via the Debug button directly above the `SCENARIO` statement
5. Results in the following unexpected behavior:
```
Filters: Scenario: s can be serialized
No test cases matched 'Scenario: s can be serialized'
===============================================================================
No tests ran
```

Note that clicking the `Run` button instead of `Debug` works fine. I suspect this is likely due to the shell parsing the back-ticks and evaluating them before actually running the command. 

I would have included the logs, but I could not find them.

**Desktop**

- Extension Version `2.7.6`
- VS Code Version `1.40.1`
- Catch2 Version `2.11.0`
- MacOS 10.14.6, but actual code and extension running in an Ubuntu 18.04 docker container

**Comment by matepek** [comment]
I guess you meant:
```cpp
SECTION("Config`s can be serialized") ...
```

**Comment by TheButlah** [comment]
Sorry I fixed my original, I meant Config to be surrounded by backticks

**Comment by matepek** [comment]
Also you have to wrap your "SECTION" with "TEST_CASE", right?

I tried to reproduce without docker, but I couldn't.
But I think I understand the problem.
Backtick has a special meaning for bash.
The adapter does everything fine. When it comes to the debugging it passes the parameters to the debug adapter through an interface. No escaping should be done there if you ask me. 
My theory is that the vscode debug adapter on ubuntu uses bash to debug / to start a debug session. And it does not escape variables. 

This is actually a security issue of the debug adapter: code can be executed remotely like SECTION("`pwd`").

I will try to escape it but I'm not sure it will work. Will see.
But this is just a workaround. the proper solution is the debug adapter to be fixed.

**Comment by TheButlah** [comment]
Hi, sorry if my example was wrong, I usually use the BDD syntax but I adapted my example to be simpler. I agree that that is probably the issue, although I'm not sure its a security issue, since to run the test you already are executing "arbitrary" code

**Comment by matepek** [comment]
You say something. Unless the debugger process has higher privileges.. (usually not)
Anyhow I am adding escaping.

**Comment by matepek** [comment]
fixed "something" in v2.7.7
Might solves this
